CHARTER.md

```diff
@@ -5,15 +5,85 @@
-|**5. Relationship to Other Projects**  |The ODF Advocacy Open Project can be related to other open source projects using ODF as their native file format such as Calligra, AbiWord and Gnumeric, or compatible with ODF such as Scribus.<br/>The ODF Advocacy Open Project can also be related to proprietary applications supporting ODF, provided they do not detract from the project’s original objective to promote the use of ODF.|
-|**6. Repositories and Licenses** |https://github.com/ODF-Advocacy Documents will be released under CC license Attribution 4.0 International (CC BY 4.0). |
-|**7. Initial Contributions from Existing Work**  |  |
+| **2. Abstract**   | 
+The objective of the PublicCode.yml Open Project is to
+develop the file format for describing public software, create awareness and
+educate the world about the benefits of using open source software in
+governments. The group will maintain the file format and keep it aligned with
+the needs of the governments and the agencies. |
+| **3. Purpose and Scope**  | 
+Public software is becoming more and more relevant each day, however its
+discoverability is crucial in order to effectively reuse it also in contexts
+different from the ones where it was originated. 
+As such, creating catalogues where public software is indexed and easily
+searchable is key to stimulate its reuse. However, catalogues need to be
+populated in an automated way to avoid scalability issues and to quickly grow.
+The publiccode.yml format goes exactly in this direction. In
+fact, it is a file that has to be stored in the root of the repository
+of the open source software in question containing the metadata
+definition of such software. In this way, it is quite trivial for automated
+algorithms, such as a crawler, to look for the file, extract relevant
+information from it, create a HTML page dedicated to that software and then insert
+such page in the catalogue.
 
-For more information, please see:
-- [[Open Projects Guide]](https://github.com/oasis-open-projects/documentation/tree/master/guides)
-- [[Getting Started with Open Projects]](https://github.com/oasis-open-projects/documentation/blob/master/guides/getting-started-guide.md)
-- [[Open Project Rules]](https://www.oasis-open.org/policies-guidelines/open-projects-process)
-- [[Open Projects Website]](http://oasis-open-projects.org)
+<br/><br/>The following is an initial list of activities and materials
+which may be developed by the publiccode.yml Open Project working group:<br/><br/>
+<ol>
+    <li>**Standard Development
```

Comment:
I believe this (Standard development) should be the real deal. We should clearly express this intention.